stdlib/asm/collections/mmr.masm

```diff
@@ -0,0 +1,100 @@
+#! Computes the `ilog2(number)` and `2^(ilog2(number))`.
+#!
+#! number must be non-zero, otherwise this will error
+#!
+#! Stack transition:
+#! Cycles:  12 + 9 * leading_zeros
+#! Input: [number, ...]
+#! Output: [ilog2, power_of_two, ...]
```

Comment:
minor nit: let's put cycles after "output" and also let's add extra space on the input line to make it look something like this:
```
#! Stack transition:
#! Input:  [number, ...]
#! Output: [ilog2, power_of_two, ...]
#!
#! Cycles:  12 + 9 * leading_zeros
```

Reply:
moved cycles below input/output